### ISSUE TITLE
stylus is not minimized in production mode

### ISSUE THREAD

**Issue by terion-name** [JeffreyWay/laravel-mix#427]
with `npm run production` stylus output is not minimized

**Comment by JeffreyWay** [comment]
I just tested it, and they are being minified. 

https://www.dropbox.com/s/fwvfo3a7rg1xlaq/Screenshot%202017-02-20%2009.51.23.png?dl=0

Can you give me reproducible steps?

**Comment by terion-name** [comment]
Hm. All the stuff:

mix (using latest version directly from github, btw)
```js
const mix = require('laravel-mix');

mix.options({
  //extractVueStyles: true
});

const stylusOptons = {
  paths: ['node_modules', 'resources/assets/stylus'],
  'include css': true,
  'resolve url': true
};

mix.setPublicPath('./public/assets');

mix.stylus('resources/assets/stylus/home.styl', 'css', stylusOptons);

mix
    .stylus('resources/assets/stylus/vote/main.styl', 'css/vote', stylusOptons)
    .stylus('resources/assets/stylus/vote/alter.styl', 'css/vote', stylusOptons)
    .stylus('resources/assets/stylus/vote/order.styl', 'css/vote', stylusOptons)
    .stylus('resources/assets/stylus/vote/open.styl', 'css/vote', stylusOptons);

mix.js('resources/assets/js/vote-ui/order.js', 'js/vote');

mix
    .sourceMaps()
    .js('resources/assets/js/app.js', 'js')
    .js('resources/assets/js/ui.js', 'js')
    .extract(['vue', 'vuex', 'vue-resource', 'vue-router', 'laravel-echo', 'socket.io-client', 'vue-multiselect', 'isotope', 'vueisotope', 'isotope-packery', 'vue-chartjs', 'vue-color', /*'vue-awesome',*/ 'vue-paginate'])
    .stylus('resources/assets/stylus/app.styl', 'css', stylusOptons)
    .copy('resources/assets/images', 'public/assets/images', false)
    .copy('resources/assets/fonts', 'public/assets/fonts', false);
```

Clearing `public/assets/css`, running `npm run production`, checking `public/assets/css/app.css`:

<img width="587" alt="2017-02-20 16 58 38" src="https://cloud.githubusercontent.com/assets/1060205/23130098/d937a9e6-f78d-11e6-80bc-f6d8a5a9289d.png">

npm scripts:
```json
"scripts": {
    "dev": "cross-env NODE_ENV=development webpack --progress --hide-modules",
    "watch": "cross-env NODE_ENV=development webpack --watch --progress --hide-modules",
    "hot": "cross-env NODE_ENV=development webpack-dev-server --inline --hot",
    "production": "cross-env NODE_ENV=production webpack --progress --hide-modules",
    "echo": "laravel-echo-server start"
  }
```

webpack.config: 
[webpack.config.js.zip](https://github.com/JeffreyWay/laravel-mix/files/787805/webpack.config.js.zip)

**Comment by JeffreyWay** [comment]
I've tried a few variations, and it minimizes properly every time. 

Please try to break this down into a reproducible set of steps from scratch. It's difficult to debug an existing big build setup.

**Comment by JeffreyWay** [comment]
Okay, I did a little more digging and it seems to be related to the `mix.options()` call. If you delete that entirely, it should minify. I'll figure out what the issue is there. 

Either way, it's related to your issue about extracting Vue styles. So I'll close this one.

**Comment by terion-name** [comment]
@JeffreyWay true! without options minification works. weird thing)

**Comment by JeffreyWay** [comment]
This is fixed in ecddc5e859709cb3cfd3b6e00bc1320da24c3690

**Comment by terion-name** [comment]
confirm, thank you)
